data_steward/tools/load_vocab.sh

```diff
@@ -12,12 +12,14 @@ tools/load_vocab.sh --app_id app_id --in_dir in_dir [--gcs_path gcs_path --datas
  --in_dir   in_dir   directory where vocabulary files are located
  --gcs_path gcs_path full GCS path to save transformed files
  --dataset  dataset  name of BigQuery dataset to create and load vocabulary
+ [--venv_dir venv_dir directory where virtual environment should be created]
```

Comment:
Is this a major difference between command line development and PyCharm development?  From my perspective, it's redundant.

If you create a virtual environment, consider cleaning it up before leaving the script with a good 
```
deactivate
rm -rf ${VENV_DIR}
```

Reply:
Does it seem redundant because the script can assume a virtual environment will be activated prior to running it? Or because the location of the virtual environment should be assumed? I noticed most of our scripts assume a virtual env in `${CURATION_HOME}/dataset_steward/curation_venv`- should I run on this assumption?

Reply:
It's redundant (for me) from a command line perspective because I always have my virtual environment activated.  That's why I was asking if this is a development environment issue (because I think I'm the only one using command line and noticing this).  If I were going to leave this in here, I wouldn't have it build the virtual environment inside our `data_steward` directory.  This is our package and product.  It's not where I would install third party libraries just because someone might forget to clean it up.

Reply:
I agree- I've had issues in the past when the virtual env was in `data_steward`. Like you I run these from command line myself but yesterday I'd forgotten to activate my environment first. The terminal alerted me there was a problem only after this script had run for several minutes; so I put the virtual env stuff in to prevent wasted time. Perhaps this is a good topic for team discussion. For now I can do the clean up you suggested if that's ok with you.

Reply:
Yes, please do the clean up.